sdk/react/src/lib/fragment-components/a-inline-notification.tsx

```diff
@@ -0,0 +1,202 @@
+`;
+
+export const AInlineNotificationSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	const kind = [
+		{ id: 'error', text: 'Error' },
+		{ id: 'info', text: 'Info' },
+		{ id: 'info-square', text: 'Info square' },
+		{ id: 'success', text: 'Success' },
+		{ id: 'warning', text: 'Warning' },
+		{ id: 'warning-alt', text: 'Alternative warning' }
+	];
+
+	return <>
+		<Checkbox
+			labelText='Hide close button'
+			id='hide-close-button'
+			checked={selectedComponent.closeButtonHidden}
+			onChange={(_: any, { checked }: any) => {
+				setComponent({
+					...selectedComponent,
+					closeButtonHidden: checked
+				});
+			}} />
+		<Checkbox
+			labelText='Low contrast'
+			id='low-contrast'
+			checked={selectedComponent.lowContrast}
+			onChange={(_: any, { checked }: any) => {
+				setComponent({
+					...selectedComponent,
+					lowContrast: checked
+				});
+			}} />
+		<Dropdown
+			id='notification-kind'
+			label='Notification kind'
+			titleText='Notification kind'
+			items={kind}
+			selectedItem={kind.find(item => item.id === selectedComponent.kind)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					kind: event.selectedItem.id
+				});
+			}} />
+		<TextInput
+			light
+			value={selectedComponent.title}
+			labelText='Title'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				title: event.currentTarget.value
+			})} />
+		<TextInput
+			light
+			value={selectedComponent.subtitle}
+			labelText='Subtitle text'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				subtitle: event.currentTarget.value
+			})} />
+	</>;
+};
+
+export const AInlineNotificationCodeUI = ({ selectedComponent, setComponent }: any) => {
+	return <TextInput
+		id='input-name'
+		value={selectedComponent.codeContext?.name}
+		labelText='Input name'
+		onChange={(event: any) => {
+			setComponent({
+				...selectedComponent,
+				codeContext: {
+					...selectedComponent.codeContext,
+					name: event.currentTarget.value
+				}
+			});
+		}}
+	/>;
+};
+
+export const AInlineNotification = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+		{...rest}>
+			<InlineNotification
+				className={cx(preventCheckEventStyle, componentObj.cssClasses?.map((cc: any) => cc.id).join(' '))}
+				kind={componentObj.kind}
+				hideCloseButton={componentObj.closeButtonHidden}
+				lowContrast={componentObj.lowContrast}
+				subtitle={componentObj.subtitle}
+				title={componentObj.title} />
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: AInlineNotification,
+	settingsUI: AInlineNotificationSettingsUI,
+	codeUI: AInlineNotificationCodeUI,
+	keywords: ['notification', 'inline', 'notify', 'alert'],
+	name: 'Inline notification',
+	type: 'inline-notification',
+	defaultComponentObj: {
+		type: 'inline-notification'
```

Comment:
```suggestion
		type: 'inline-notification',
		title: 'Information',
		subtitle: 'Something important happened.'
```

After checking the user experience in the preview, it makes sense that we include a default title and subtitle.
The properties should remain optional in the SDK and Player.